lib/perl/Genome/Model/Tools/Transcriptome/ErccMapUnaligned.pm

```diff
@@ -0,0 +1,356 @@
+package Genome::Model::Tools::Transcriptome::ErccMapUnaligned;
+
+use strict;
+use warnings;
+
+use Genome;
+use Path::Class;
+
+class Genome::Model::Tools::Transcriptome::ErccMapUnaligned {
+    is => 'Command::V2',
+    has_input => [
+        bam_file => {
+            is => 'FilePath',
+            doc => 'An aligned BAM file to a reference genome without the ERCC transcripts spiked in.',
+        },
+        ercc_fasta_file => {
+            is => 'FilePath',
+            doc => 'The FASTA format sequence for all ERCC transcripts.',
+            example_values => ['/gscmnt/gc13001/info/model_data/jwalker_scratch/ERCC/ERCC92.fa'],
+        },
+        ercc_spike_in_file => {
+            is => 'Text',
+            doc => 'The control analysis file provided by Agilent for the ERCC spike-in.',
+            example_values => ['/gscmnt/gc13001/info/model_data/jwalker_scratch/ERCC/ERCC_Controls_Analysis.txt'],
+        },
+        ercc_spike_in_mix => {
+            is => 'Integer',
+            doc => 'The expected ERCC spike-in mix.',
+            valid_values => [ '1', '2'],
+        },
+        samtools_version => {
+            is => 'Text',
+            doc => 'The version of samtools to run analysis.',
+            example_values => ['1.2'],
+            default_value => '1.2',
+            is_optional => '1',
+        },
+        samtools_max_mem => {
+            is => 'Integer',
+            doc => 'The max memory required by samtools (in bytes).',
+            example_values => ['14000000000'],
+            default_value => '14000000000',
+            is_optional => '1',
+        },
+        bowtie2_version => {
+            is => 'Text',
+            doc => 'The version of bwa to run analysis.',
+            example_values => ['2.1.0'],
+            default_value => '2.1.0',
+            is_optional => '1',
+        },
+    ],
+    has_output => [
+        pdf_file => {
```

Comment:
I was going to suggest it be a `FilePath` instead of `Text` so went to look for how it was used but I don't see this being used anywhere in the code.